pkg/systemd/logs.js

```diff
@@ -399,6 +409,167 @@ $(function() {
                     .replaceWith(new_content);
         }
 
+        function set_up_prompt_dialog(path, message, type) {
+            var dialog = $('#prompt-dialog');
+            let body = dialog.find('.modal-body');
+            var input = $('<input class="form-control">');
+            let footer = dialog.find('.modal-footer');
+
+            body.empty().append($('<p></p>').text(message));
+            footer.empty();
+
+            // Content widgets
+            switch (type) {
+            case PromptType.ASK_PASSWORD: {
+                input.attr("type", "password");
+            }
+            // falls through
```

Comment:
This comment is redundant. Same for all comment with same string..

Reply:
It really is not:
```
/home/ekulik/Development/github.com/cockpit-project/cockpit/pkg/systemd/logs.js
  487:13  error  Expected a 'break' statement before 'case'  no-fallthrough
```

Reply:
Oh sorry, probably it's some eslint rule we had disabled all this time.